### ISSUE TITLE
Implement Static Picklists

### ISSUE THREAD

**Issue by gsteimer** [tsgrp/hpi#694]
Currently, if you try to create a static picklist in HPI Admin, the following is shown to the user:

![image](https://cloud.githubusercontent.com/assets/1377228/7961335/8037942e-09cc-11e5-8686-290105bbbd8e.png)

This enhancement would add in static picklists to HPI admin, allowing administrators to manage the picklist from HPI.

- [x] Implement picklist backend - is this just the JSON picklists that were added for a client last year?
- [x] HPI Admin
  * User can add picklists
  * User can manage (add, remove, modify list items)
- [x] Ensure that form support works with the new picklist type

**Comment by gsteimer** [comment]
Also - this really isn't a static picklist.  should we rename this to something like "HPI Picklist"?

**Comment by tbrandstatter** [comment]
Implemented with revision 12651

Phase 1 of Simple picklist enhancement completed. Phase two of factoring out the ko/kb code in favor of backbone still needs to be completed.

**Comment by zkeller** [comment]
Update from Taylor when she left:

So,functionality wise everything in the picklist config is working correctly (I think so--I checked a lot, but I may have missed something), so there shouldn't need to be too much if anything added to the js file.

I got most of the unit testing done, to make it easier, I highlighted the unit tests on the google doc that I didn't get to, or I couldn't figure out. (google doc: https://docs.google.com/document/d/1V5OguBhfxcpjNWI58lg7_mitsoB1qW-zuf7KrteIboM/edit)

The only concern I can really think of is if the picklist loads into a form and can be used. I haven't checked in a while, but last I did it didn't work. I think it has something to do with the picklistService in conetxt.js.
